src/Persistence/Migrations/20230604201330_BaseId.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Crpg.Persistence.Migrations;
+
+/// <inheritdoc />
+public partial class BaseId : Migration
```

Comment:
Could you give a more explicit name to the migration such as "AddBaseItemToItem" please